test/e2e/singlecluster/baseline/statefulset_test.go

```
@@ -23,6 +23,8 @@ import (
 	"github.com/onsi/gomega"
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

golangci-lint (2.12.2)

[error] dupImport: package is imported 2 times under different aliases on lines 26 and 27

(gocritic)